testConnectivity.py

```diff
@@ -937,13 +946,17 @@ def testConnectivity(args):
         else:
             printGreen("SCurve Analysis Completed Successfully")
 
-        print("| OH | N_DEAD |")
-        print("| -- | ------ |")
+        print("| OH | VFAT | N_DEAD |")
+        print("| -- | ---- | ------ |")
         tooManyDeadChan = False
-        for ohN,nDeadChan in enumerate(nDeadChanByOH):
-            print("| {0} | {1}{2}{3} |".format(ohN,colors.RED if nDeadChan > 3 else colors.GREEN,nDeadChan,colors.ENDC))
-            if nDeadChan > 3:
-                tooManyDeadChan = True
+        sumDeadChan = 0
+        for ohN,ResultsByVfat in enumerate(nDeadChanByOH):
```

Comment:
Need a better way here to set ohN, this will only be correct if `nDeadChanByOH` includes all 12 OH's and completes in order...